src/ring/server/standalone.clj

```diff
@@ -99,7 +99,7 @@
          (addShutdownHook (Thread. destroy))))
     (try-port (port options)
       (fn [port]
-        (let [options (merge {:port port} options)
+        (let [options (assoc options :port port)
```

Comment:
This overwrites what's in options, but that seems correct, since the value of port was determined by looking at options in https://github.com/PawelStroinski/ring-server/blob/b3e1bc2354decd7181f85c079d27606d9709cdf5/src/ring/server/options.clj#L9